packages/react-renderer-demo/src/pages/custom-examples/sample-example.md

```diff
@@ -0,0 +1,20 @@
+import DocPage from '@docs/doc-page';
+import CodeExample from '@docs/code-example';
+
+<DocPage>
+
+
+# Sample example
+
+## Use markdown syntax to write text
+
+## What to include?
+- description of the component
+- what issue it can help solve
+- include component preview in a form
```

Comment:
Maybe I would use this page to say: 

"Hey if you need custom component, use useField and check this example"

We can put the text you used to `development setup` page. :thinking:

Reply:
I fully expect this example to get deleted. This is here just for @jinxac so he knows where to pud his demo.